SetupTearDown.ps1

```diff
@@ -9,14 +9,26 @@
 .PARAMETER Cleanup
     This switch triggers the cleanup path which disables WinRM service, removes the firewall rules that were created earlier for remoting, and also Disables PSRemoting
 
+.PARAMETER SetupContainerHost
+    This switch triggers the setup calls which end up enabling PS Remoting over SSH on the container host machine (generating a public key and installing PowerShell 7)
+
+.PARAMETER SetupContainer
+    This switch triggers the setup calls which end up enabling PS Remoting over SSH on a container (installing and configurating OpenSSH server and PowerShell 7)
+
+.PARAMETER AuthorizedKey
+    The SSH public key generated as part of the SetupContainerHost call. Should be passed to SetupContainer calls for OpenSSH configuration.
+
 .DESCRIPTION
     Run this script to setup your machine for PS Remoting so that you can leverage the functionality of runPerfTool.psm1
     Run this script at the end of the tool runs to restore state on the machines.
     Ex: SetupTearDown.ps1 -Setup or SetupTearDown.ps1 -Cleanup
 #>
 Param(
-    [switch] $Setup,
-    [switch] $Cleanup
+    [Parameter(Mandatory=$True, ParameterSetName="Setup")]              [switch] $Setup,

```

Comment:
These should not be mandatory (doing so would require you to specify -Cleanup:$false)